src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageAndTag.cs

```diff
@@ -99,7 +99,7 @@ public ComputeDotnetBaseImageAndTag()
         TargetFrameworkVersion = "";
         ContainerFamily = "";
         FrameworkReferences = [];
-        TargetRuntimeIdentifier = "";
+        TargetRuntimeIdentifiers = Array.Empty<string>();
```

Comment:
nit: The new empty collection initialization is more brief and expressive.

```suggestion
        TargetRuntimeIdentifiers = [];
```